src/tools/meson-flex-wrapper

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--flex', dest='flex_command', help='flex executable to run', type=str)
+parser.add_argument('--perl', dest='perl_command', help='perl executable to run', type=str)
+parser.add_argument('--fix_warning_script', dest='fix_warning_script', help='script for fixing warnings', default='')
+parser.add_argument('-o', dest='output_file', help='output file')
+
+args, rest = parser.parse_known_args()
+rest_flags = ' '.join(rest)
+
```

Comment:
We should change to a private directory ("@PRIVATE_DIRECTORY@") or such before running flex, because otherwise the lex.backup locations conflict. Unfortunately the parameter to control the location of lex.backup only was added in an as-of-yet-unreleased (but used in debian) version of flex.